py-polars/polars/interchange/protocol.py

```diff
@@ -255,3 +257,45 @@ class Endianness:
 
 class CopyNotAllowedError(RuntimeError):
     """Exception raised when a copy is required, but `allow_copy` is set to `False`."""
+
+
+class Flavor:
```

Comment:
I am not really happy with calling this `Flavor`. This was an internal name.

Can you call this `CompatLevel`?